src/components/GradeItemAccordion.tsx

```diff
@@ -109,7 +123,11 @@ function GradeItemAccordion({ item, updateItem }: GradeItemAccordionInterface) {
 						</RowComponent>
 					</AccordionExtendedComponent>
 					<AccordionExtendedComponent>
-						<RowComponent>
+						<RowComponent
+							style={{
+								gridArea: 'weight',
+							}}
+						>
```

Comment:
Did the formatter put this on many lines?

Reply:
If you even wanted to omit the lengthly style thing, you could do something like make the gridArea a prop on the component. Checkout the docs here:

https://styled-components.com/docs/basics#passed-props